src/index.html

```diff
@@ -11,7 +11,167 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="section">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="surname"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+            name="name"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            autocomplete="off"
+            value="12"
+            min="1"
+            max="100"
+            name="age"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="birthDate"
+          >
+        </label>
+
+        <label class="form-field, field" required>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="termAccepted"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="section">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+            name="eMail"
+          >
+        </label>
+
+        <label class="form-field, field">
+          Password:
+          <input
+            type="password"
+            autocomplete="off"
+            minlength="5"
+            maxlength="32"
+            required
+            name="password"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="section">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+
+          <label>
+            <input type="radio" name="loveCats">
+              Yes
+          </label>
```

Comment:
These are the same level elements (text inside label and input), there's no need for additional indents
```suggestion
          <label>
            <input type="radio" name="loveCats">
            Yes
          </label>
```